modules/chat/src/routes/index.ts

```diff
@@ -538,23 +590,40 @@ export class ChatRoutes {
       {
         path: '/leave/:roomId',
         action: ConduitRouteActions.UPDATE,
-        description: `Removes current user from a chat room.`,
+        description: `Context user leaves chat room.`,
```

Comment:
Context doesn't mean anything to an API user. If you don't like "current" change it to something that the devs will understand. The behaviour is that the user making the request will leave.